lib/facter/fpm_conf_d.rb

```diff
@@ -0,0 +1,11 @@
+Facter.add(:fpm_conf_d) do
+  confine :osfamily => 'Debian'
+
+  setcode do
+    Dir.glob('/etc/php5/fpm/conf.d/*.ini').each do |dot_config|
```

Comment:
it will not work on RetHat distros as set on params.pp, the fpm configuration leaves at:
/etc/php-fpm.d/